deploy-on-k8s.yml

```diff
@@ -13,14 +13,18 @@ spec:
       labels:
         app: ci
     spec:
+      securityContext:
+        runAsUser: 65534
```

Comment:
This is "nobody". The reason I need to do this is there is no (automated) way to predict the UID that the container would otherwise run as, and therefor no way to know what to put in /etc/subuid in order to allow building with a user other than root.